app/models/in_person_enrollment.rb

```diff
@@ -134,21 +141,20 @@ def on_notification_sent_at_updated
   end
 
   def eligible_for_notification?
-    self.notification_phone_configuration.present? &&
-      (self.passed? || self.failed? || self.expired?)
+    self.notification_phone_configuration.present? && (self.passed? || self.failed?)
   end
 
   private
 
   def on_status_updated
-    if enrollment_will_be_cancelled? && notification_phone_configuration.present?
+    if enrollment_will_be_cancelled_or_expired? && notification_phone_configuration.present?
       notification_phone_configuration.destroy!
     end
     self.status_updated_at = Time.zone.now
   end
 
-  def enrollment_will_be_cancelled?
-    status_change_to_be_saved&.last == 'cancelled'
+  def enrollment_will_be_cancelled_or_expired?
+    [STATUS_CANCELLED, STATUS_EXPIRED].include? status_change_to_be_saved&.last
```

Comment:
What is `status_change_to_be_saved`? Is that some ActiveRecord thing showing staged changes yet to be committed to the db? I didn't see anything on a quick google search

Reply:
yeah, it's [ruby magic](https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Dirty.html#method-i-attribute_change_to_be_saved). You can use `<name>_change_to_be_saved` to get the value for any variable that's being updated.